src/utils/OptionParser.cpp

```diff
@@ -345,11 +346,13 @@ void OptionParser::process_opt(const Option& o, const std::string& opt, const st
 	}
 	else if (o.action() == "help") {
 		print_help();
-		mardyn_exit(0);
+		std::ostringstream empty_message;
+		mardyn_exit(empty_message, __FILE__, __LINE__, EXIT_SUCCESS);
 	}
 	else if (o.action() == "version") {
 		print_version();
-		mardyn_exit(0);
+		std::ostringstream empty_message;
+		mardyn_exit(empty_message, __FILE__, __LINE__, EXIT_SUCCESS);
 	}
```

Comment:
This is not very elegant. The problem is that ls1 does not have any proper finishing method. Until now, just `MPI_Abort` was called but with exit code `0`. Maybe call `MPI_Finalize` and then `exit(0)` here, similar to the last lines in `Mardyn.cpp` ? Any other idea, @FG-TUM ? I am not sure if `exit()` does a proper cleanup, there are opposite information on the internet.

Reply:
I have done this now in this commit:
https://github.com/ls1mardyn/ls1-mardyn/pull/341/commits/1b363c82b6453ebb1321b4d9797624466f91cbd7